examples/demo-app/src/utils/url.js

```diff
@@ -66,3 +66,11 @@ parseUri.options = {
     loose: /^(?:(?![^:@]+:[^:@\/]*@)([^:\/?#.]+):)?(?:\/\/)?((?:(([^:@]*)(?::([^:@]*))?)?@)?([^:\/?#]*)(?::(\d*))?)(((\/(?:[^?#](?![^?#\/]*\.[^?#\/.]+(?:[?#]|$)))*\/?)?([^?#\/]*))(?:\?([^#]*))?(?:#(.*))?)/
   }
 };
+
+/**
+ * Validates an url
+ * @param str
+ */
+export function validateUrl(str) {
+  return str?.match(/^(ftp|http|https?):\/\/+(www\.)?[a-z0-9\-\.]{3,}\.[a-z]{3}$/);
```

Comment:
This regex only allows URLs with an empty path (allows `https://uber.com` but not `https://uber.com/data.csv`).

Additionally, it removes the option of using localhost as a domain.
And TLDs with only 2 letters (.es, .cl, etc).

I think this PR should be reverted

Reply:
@jagerts @heshan0131